apps/emqx/src/emqx_ssl_crl_cache.erl

```diff
@@ -205,6 +207,7 @@ http_lookup(URL, Rest, CRLDbInfo, Timeout) ->
     end.
 
 http_get(URL, Rest, CRLDbInfo, Timeout) ->
+    ?SLOG_THROTTLE(debug, #{msg => fetching_crl_on_the_fly, url => URL}),
```

Comment:
- iirc, throttling is not done at debug level.
- does it really need throttling, it's only fetched when catch missed, right?
- maybe change `on_the_fly` to `due_to_cache_miss`

Reply:
> iirc, throttling is not done at debug level.

Indeed, that seems to be case...  :melting_face: 

> does it really need throttling, it's only fetched when catch missed, right?

If there are several clients trying to connect but the fetch fails, it'll spam the logs.

> maybe change on_the_fly to due_to_cache_miss

:checkered_flag: